engine/apps/user_management/migrations/0018_auto_20231115_1206.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2.20 on 2023-11-15 12:06
+
+import apps.user_management.utils
+from django.db import migrations, models
+
+import django_migration_linter as linter
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('user_management', '0017_alter_organization_maintenance_author'),
+    ]
+
+    operations = [
+        linter.IgnoreMigration(),
+        migrations.AddField(
+            model_name='organization',
+            name='alert_group_table_columns',
+            field=models.JSONField(default=apps.user_management.utils.default_columns),
+        ),
```

Comment:
does this field have to be non-nullable? maybe it makes sense to make it nullable?

```python
models.JSONField(null=True, default=apps.user_management.utils.default_columns)
```

Reply:
In this case migration will set value `None` instead of default for existing organizations

Reply:
Oh, got it, thanks. It makes sense to add lazy update for organization settings then, since default value works incorrectly with nullable field